lib/puppetserver/ca/cli.rb

```diff
@@ -64,8 +64,9 @@ class Cli
 
 
       def self.run(cli_args = ARGV, out = STDOUT, err = STDERR)
-        logger = Puppetserver::Ca::Logger.new(:info, out, err)
         parser, general_options, unparsed = parse_general_inputs(cli_args)
+        logger = general_options.key?('verbose') ? Puppetserver::Ca::Logger.new(:debug, out, err)
```

Comment:
Some style ideas:
1. Nothing else needs to read the `verbose` key out of these options so you might want to get the value while removing it. You can do that with `delete()`, eg `general_options.delete('verbose')`
2. I think it's cleaner to put the value of the log level in the conditional and then pass that into the Logger's constructor, eg
```rb
level = general_options.key?('verbose') ? :debug : :info
logger = Puppetserver::Ca::Logger.new(level, out, err)
```

Reply:
Right on. I'll make these changes.